uaclient/tests/test_cli_status.py

```diff
@@ -75,8 +74,7 @@
 Technical support level: n/a
 """
 
-# TODO-ROS add "ros" and "ros-updates" to BETA_SVC_NAMES
-BETA_SVC_NAMES = ["cc-eal", "esm-apps"]
+BETA_SVC_NAMES = ["cc-eal", "esm-apps", "ros", "ros-updates"]
```

Comment:
Why are we claiming ros and ros-updates are beta? Wouldn't this be the time we drop the is_beta = True definition and just make this a fully participating service?